### DISCUSSION TITLE
Why is CSV Import feature deprecated?!?!

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#8959] [General]
This feature is a major part of a few of our clients work flow. These clients have no API capabilities and use this tool regularly.
Will the 2 recommended solutions be ready to handle GUI uploading and importing of data? Are there new features coming to data-hub to handle this?

This is going to make for some VERY difficult discussions with these clients.

**Comment by brusch** [comment]
See https://github.com/pimcore/data-importer
